### ISSUE TITLE
How to add Finnish to target translator? 

### ISSUE THREAD

**Issue by XB-Pixel** [ogkalu2/comic-translate#170]
I would like to know how so I can add it myself.
What to edit and add?
I would be thankful if you added yourself, but I can also do it if I get help.

**Comment by albar18** [comment]
It's quite simple, just open the commits page and look for commits that contain the phrase "add target translation" in there you will find which files need to be modified. Just follow the writing format for each file that is changed. For the language code you want to add, please search for it on Google.

**Comment by XB-Pixel** [comment]
I'm thankful for the help Albar18, but can you explain it more simply sorry I'm new coding where to go and what file to edit.
